uber/templates/macros.html

```diff
@@ -119,13 +119,13 @@
   </div>
 {%- endmacro %}
 
-{% macro address_form(group=False, attendee=False, is_prereg_dealer=False, copy_address=False, name_prefix="") -%}
+{% macro address_form(group=False, attendee=False, is_dealer=False, copy_address=False, name_prefix="") -%}
     {% set group_or_attendee=group if group else attendee %}
     {% set id_prefix=group.id|replace("-","") if group else attendee.id|replace("-","") %}
-    {% if not is_prereg_dealer %}
-        <script src="../static/deps/selectToAutocomplete/jquery.select-to-autocomplete.js"></script>
-        <script type="text/javascript">{% include "region_opts.html" %}</script>
-    {% endif %}
+
+    <script src="../static/deps/selectToAutocomplete/jquery.select-to-autocomplete.js"></script>
```

Comment:
Okay, so here's the thing, we gotta print these every time the macro is called. There's no good way to determine if they've already been included.